### DISCUSSION TITLE
Why not keep branches online?

### DISCUSSION THREAD

**Discussion by Angra974** [cameronapak/freedom-stack#33] [General]
Hi. Previously, you were using Clerk to bootstrap the login of the stack. Now, it has changed to BetterAuth. No problem with this. But with Clerk, just a few clicks and it was done, and we could test. Loved it that way. I haven't tested BetterAuth yet, as I'm still digging around, so maybe later.

Previously, you were using TinyMCE and custom JavaScript to have it in the public folder for access when using the application. Now, you've changed it to Trix. Again, no problem with this. But some users might want to use TinyMCE instead of Trix or Clerk instead of BetterAuth.

So, why not save them in branches and keep the code usable for those who want to see how it was implemented or how to implement it?

This way, you could have: v0.1.1 with Clerk, v0.1.2 with TinyMCE (no more updates?), and the current main application with the current updates. What do you think about this?

**Comment by cameronapak** [comment]
Hey, thank you! I never want anyone to feel a rug pulled out from under them. Trust erodes in that way, so I apologize if these changes brought confusion or stress in any way.

I'm new to creating starter kits, boilerplates, and npm packages. I want to make a way where someone can update once they've created their own project from Freedom Stack, but I haven't yet found a way to do that **(I'm open to ideas!)**. 

In the [Releases](https://github.com/cameronapak/freedom-stack/releases), you may be able to use the previous versions. Or, I'd encourage you to fork the codebase and then move the HEAD to the latest commit before removing Clerk or TinyMCE. 

Going forwards, I will do my best to make sure there are no rug pulls. 

I want to support Clerk, LogTo, and Better Auth. However, I want that in a script, so that it's easy to choose the auth you want. _(Please feel free to create a PR for that!)_

I appreciate you, @Angra974, and your feedback. Where would you like to see Freedom Stack build towards?

**Reply by cameronapak** [reply]
However, I did find an old version available at https://github.com/jfleagl12/freedom-stack

**Reply by Angra974** [reply]
I've lot of version in my computer so it's not a problem. Just for people that are not use to the application and find it for the first time and know about clerk or need to use clerk and find the stack useful. I know some repositories do like that with branches so i think it's not a bad idea to keep it, or update the docs to link to the release for "If you want to use clerk for authentification, check our previous release X and see how it was implemented in our application" so it's easier to find and check it. They will benefit more this way.

**Reply by cameronapak** [reply]
Yes, in the future that will be as easy as a CLI command :)

So, if I wanted to switch to clerk, I could just `npm run auth:clerk` or something like that. Hope that would help! I also plan to make tutorials, too
